doc/org.eclipse.cdt.doc.user/concepts/cdt_c_before_you_begin.htm

```diff
@@ -30,17 +30,26 @@ <h2>Windows</h2>
 MinGW produces native Windows executables that do not require a separate runtime.</p>
 
 <ul>
-<li><p>For <strong>MinGW</strong>, it is recommended to use the MinGW installer, mingw-get, to manage your
-MinGW installation. Download and run the lateset mingw-get-inst package from the
-<a href="https://sourceforge.net/projects/mingw/files">MinGW Downloads page.</a>
-The CDT MinGW toolchain will find this install if it is located in the default C:\MinGW directory, the MinGW bin
-directory is placed in your path, or if the MinGW location is stored in the MINGW_HOME environment variable.</p>
-<p>Note that the Wascana IDE has been discontinued.</p>
+<li><p>For <strong>MinGW</strong>, it is recommended to use the MSYS2 software distribution.
+The CDT will automatically detect tools that are installed using the MSYS2 package manager.
+Download and run the latest MSYS2 installer by following instructions on the <a href="https://www.msys2.org/">MSYS2 Getting Started</a> page.
+When you reach the UCRT64 environment prompt, use the following commands to install individual tools:</p>
+<p />
+<p><table style="border: 1px solid grey; border-collapse: collapse;">
+<tr><th style="border: 1px solid grey; padding: 5px;">Tool</th>   <th style="border: 1px solid grey; padding: 5px;">Installation command</th><tr>
+<tr><td style="border: 1px solid grey; padding: 5px;">clangd</td> <td style="border: 1px solid grey; padding: 5px;"><tt>pacman -S mingw-w64-ucrt-x86_64-clang-tools-extra</tt></td></tr>
+<tr><td style="border: 1px solid grey; padding: 5px;">cmake</td>  <td style="border: 1px solid grey; padding: 5px;"><tt>pacman -S mingw-w64-ucrt-x86_64-cmake</tt></td></tr>
+<tr><td style="border: 1px solid grey; padding: 5px;">gcc</td>    <td style="border: 1px solid grey; padding: 5px;"><tt>pacman -S mingw-w64-ucrt-x86_64-gcc</tt></td></tr>
```

Comment:
I recommend adding gdb too, which I think would be. mingw-w64-ucrt-x86_64-gdb 

```suggestion
<tr><td style="border: 1px solid grey; padding: 5px;">gcc</td>    <td style="border: 1px solid grey; padding: 5px;"><tt>pacman -S mingw-w64-ucrt-x86_64-gcc</tt></td></tr>
<tr><td style="border: 1px solid grey; padding: 5px;">gdb</td>    <td style="border: 1px solid grey; padding: 5px;"><tt>pacman -S mingw-w64-ucrt-x86_64-gdb</tt></td></tr>
```

Reply:
Good catch. Somehow I missed `gdb`!